PocketKit/Sources/PocketKit/Main/RegularMainCoordinator.swift

```diff
@@ -40,23 +40,46 @@ class RegularMainCoordinator: NSObject {
             .environment(\.tracker, tracker)
 
         splitController = UISplitViewController(style: .tripleColumn)
-        navigationSidebar = UIHostingController(
-            rootView: NavigationSidebarView(model: model)
-        )
-
+        splitController.displayModeButtonVisibility = .always
+        
+        navigationSidebar = UINavigationController(rootViewController: NavigationSidebarViewController(model: model))
+        
         myList = UIHostingController(rootView: listView)
+        myList.view.backgroundColor = UIColor(.ui.white1)
+        
         home = HomeViewController(source: source)
+        home.view.backgroundColor = UIColor(.ui.white1)
+        
         item = ItemViewController(
             model: model,
             tracker: tracker,
             source: source
         )
+        item.view.backgroundColor = UIColor(.ui.white1)
 
         super.init()
 
         splitController.setViewController(navigationSidebar, for: .primary)
         splitController.setViewController(myList, for: .supplementary)
         splitController.setViewController(item, for: .secondary)
+        
+        splitController.view.tintColor = UIColor(.ui.grey1)
+
+        navigationSidebar.navigationBar.prefersLargeTitles = true
+        navigationSidebar.navigationBar.barTintColor = UIColor(.ui.white1)
+        navigationSidebar.navigationBar.tintColor = UIColor(.ui.grey1)
+
+        myList.navigationController?.navigationBar.prefersLargeTitles = true
```

Comment:
kinda sad that we have to wait until these have been added to the split before we can access the nav controller. As an alternative, we could preemptively wrap `myList` and `home` in navigation controllers that we own. Not a big deal but would be a bit prettier than having to dig so far down to get to the navigation bar (and wouldn't need to unwrap the optional).

That said, it's not a huge deal so I'll leave it to your discretion.